server/api/muster/muster.controller.ts

```diff
@@ -86,12 +86,13 @@ class MusterController {
       .leftJoinAndSelect('unit.org', 'org')
       .where('json_array_length(unit.muster_configuration) > 0')
```

Comment:
I'm not really sure what the behavior of `json_array_length` is when the value is null, but it seems like the check should be WHERE `(unit.muster_configuration IS NOT NULL AND json_array_length(unit.muster_configuration) >= 0) OR (unit.muster_configuration IS NULL AND json_array_length(org.default_muster_configuration) > 0)`.  It seems like the current logic would fall back to checking the default muster configuration if the length was 0.

Reply:
The previous logic didn't return any rows that were 0 in length. The new logic aligns with that behavior but then also returns rows that have NULL if there is a default must config. So are you sure we should be using `>=` in that case? I don't fully understand what's happening here - from a business rules point of view. So let me know.

That said, I believe I can write the code to be more clear. I'll take a shot at it.